pkg/storage/gc_queue.go

```diff
@@ -751,6 +751,13 @@ func RunGC(
 	txnMap := map[uuid.UUID]*roachpb.Transaction{}
 	intentSpanMap := map[uuid.UUID][]roachpb.Span{}
 
+	const (
+		intentBudgetBytes = 30 * 1 << 20 // 30mb
```

Comment:
`intentBytesBudget` and `keyBytesBudget` sound better to me. Take it or leave it.